README.md

```diff
@@ -52,16 +52,25 @@ $ npm install jasmine-enzyme --save-dev
 
 ## Setup
 
-###### Jest
-If you are using [jest](https://facebook.github.io/jest/), the simplest setup is to use jests `setupTestFrameworkScriptFile` config. Add this to your `package.json`
+### Jest
+If you are using [jest](https://facebook.github.io/jest/), the simplest setup
+is to use jest's `setupTestFrameworkScriptFile` config. You'll also need to
+tell jest to unmock `react`, `enzyme`, and `enzyme-jest`.
```

Comment:
Oops sorry, that should indeed be `jasmine-enzyme`.

I get the following error when removing `jasmine-enzyme` from `unmockedModulePathPatterns`:

`TypeError: expect(...).toHaveText is not a function`